src/pages/api/sound-artists.ts

```diff
@@ -0,0 +1,52 @@
+import { NextApiHandler } from "next"
+import fetcher from "utils/fetcher"
+
+const fetchArtists = (searchQuery: string) =>
+  fetcher("https://api.sound.xyz/graphql", {
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      "X-Sound-Client-Key": process.env.SOUND_API_KEY,
+    },
+    body: {
+      query: `{
+        search(input: {limit: 20, text: "${searchQuery}"}) {
+          artists {
+            id
+            name
+            soundHandle
+            user {
+              avatar{
+                url
+              }
+            }
+          }
+        }
+      }`,
+    },
+  }).then((res) =>
+    res?.data?.search?.artists?.filter(
+      (artist) =>
```

Comment:
why is this JS filtering needed?

Reply:
it's needed because the api searches for every match, but i need only the soundHandle and artist name match